### ISSUE TITLE
Requested NSSAI from UE not found 

### ISSUE THREAD

**Issue by giangdh1009** [s5uishida/open5gs_5gc_ueransim_snssai_upf_sample_config#1]
I tried to replicated your sample, and it was success, but as wireshark trace, i can't find the Requested NSSAI from gnodeB, but only allowed NSSAI from AMF to gnodeB at NGAP Initial Context Setup Request message message
I attached the pcap capture at Control Plane Node.
https://1drv.ms/u/s!AraARB5vdTNrhrgDZNsQ8Kk5Z3F1wA?e=dzBl2Q 
Could you help me to clarify?

**Comment by s5uishida** [comment]
Hi @giangdh1009 

If you are using UERANSIM, I think that UERANSIM probably don't send `Requested NSSAI`.
In this case, I think that the NSSAI value registered in the UDM was used as default and sent from the 5GC to the UE as `Allowed NSSAI`.

Sorry if my guess is wrong.

**Comment by giangdh1009** [comment]
Actually i tried your instrucstion, which was success, UE with config at UERANSIM with sst: 1 sd: 1 go to UPF1, and sst: 1 sd: 2 go to UPF2, in UDM i provisioned both two slices which sst: 1 and sd: 1 default. Does your project trace have UERANSIM send Requested NSSAI.?

**Comment by s5uishida** [comment]
When I checked my pcaps, the `NGAP/NAS-5GS InitialUEMessage, Registration request` did not include the `Requested NSSAI`. On the other hand, the AMF -> UE `NGAP/NAS-5GS InitialContextSetupRequest` contained the appropriate `Allowed NSSAI` for the cases of [SST:1, SD:0x000001] and [SST:1, SD:0x000002] respectively. (My UDM defaults to [SST:1, SD:0x000001].)

This also happened in the following scenario with free5GC+UERANSIM.

https://github.com/s5uishida/free5gc_ueransim_snssai_upf_sample_config

**Comment by giangdh1009** [comment]
Yes, it's the same for me, So my question here is how AMF know to send the appropriate Allowed NSSAI on NGAP/NAS-5GS InitialContextSetupRequest ?

**Comment by s5uishida** [comment]
I would like to know it, too!

**Comment by giangdh1009** [comment]
Yeah, just think your scenario sample so you should know, i will try to investigate more, thanks for your support

**Comment by s5uishida** [comment]
Hi @giangdh1009 

It was solved by the following.

https://github.com/open5gs/open5gs/discussions/1703
